generated client and sanitations added (#3)
e19166/module-ballerinax-openai-audio: docs/spec/sanitations.md

_Authors_: @e19166 \
_Created_: 2024/08/05 \
_Updated_: 2024/08/05 \
_Edition_: Swan Lake

# Sanitation for OpenAPI specification

This document records the sanitation done on top of the official OpenAPI specification from OpenAI. The OpenAPI specification is obtained from the [OpenAI OpenAPI Documentation](https://github.com/openai/openai-openapi/blob/master/openapi.yaml). These changes are implemented to enhance the overall usability and readability of the generated client.

> 1. **Removed the `default:null` property of certain schemas**:
> 
>    - **Changed Schemas**: `CreateCompletionRequest`,`ChatCompletionStreamOptions`,`CreateChatCompletionRequest`
> 
>    - **Original**:
>       - defaullt: `null`
> 
>    - **Updated**:
>       - Removed the `default` parameter 
> 
>    - **Reason**: This change is done as a workaround for ballerina openapi tool not allowing to generate the client.

Reviewer: ```suggestion
1. **Removed the `default:null` property from the below schemas**:

   - **Changed Schemas**: `CreateCompletionRequest`,`ChatCompletionStreamOptions`,`CreateChatCompletionRequest`

   - **Original**:
      - default: `null`

   - **Updated**:
      - Removed the `default` parameter 

   - **Reason**: This change is done as a workaround for ballerina openapi tool not allowing to generate the client.
```
